src/azul/__init__.py

```diff
@@ -275,7 +275,7 @@ def manifest_kms_alias(self) -> str:
         # KMS requires that aliases start with '/alias'
         return 'alias/' + self.qualified_resource_name(self.manifest_kms_key_tf_name)
 
-    audit_log_retention_days = 180  # FedRAMP mandates 90 days
+    audit_log_retention_days = 365  # FedRAMP mandates 90 days
```

Comment:
The comment is a bit odd and redundant, in my opinion. Perhaps we should exhaustively list the compliance mandates being met or get ride of it.